### DISCUSSION TITLE
How change font family

### DISCUSSION THREAD

**Discussion by Ahmad-Fathy** [filamentphp/filament#4330] [General]
I tried to change font family in config file 
from:
'google_fonts' => 'https://fonts.googleapis.com/css2?family=DM+Sans:ital,wght@0,400;0,500;0,700;1,400;1,500;1,700&display=swap',

to:
'google_fonts' => 'https://fonts.googleapis.com/css2?family=Cairo:wght@200;300;400;500;600;700;800;900&display=swap',

but it didn't work

**Comment by Z3d0X** [comment]
You'll still need to set the default font as that from `tailwind.config.js`

**Reply by zepfietje** [reply]
- https://filamentphp.com/docs/2.x/admin/appearance#loading-google-fonts
- https://tailwindcss.com/docs/font-family#customizing-your-theme
- https://filamentphp.com/docs/2.x/admin/appearance#building-themes
